### PR TITLE
Add Outtake support in Teleop

### LINKED ISSUE #41: Add Teleop Outtake support
Outtake should run while a button is held by the secondary driver. Implement an existing command for the outtake, don't create a new command class. Constant for the speed should be contained within a new nested class in the constants file.

### PR DESCRIPTION
**Pull request by chit-sudeep**
Adds code to add outtake support in teleop, and adds outtake speed constant and buttonId constant.

Closes #41

### CODE REVIEW

File: src/main/java/frc/robot/Constants.java [hunk 1]
```diff
@@ -16,6 +16,8 @@
 public final class Constants {
   public static final double SIMPLE_AUTON_SPEED = 0.2; // PLACEHOLDER
   public static final double SIMPLE_AUTON_RUNTIME = 5.0; // PLACEHOLDER
+  public static final double OUTTAKE_SPEED = 5.0; // PLACEHOLDER
```
Comment (aidan-mundy) [inline src/main/java/frc/robot/Constants.java]: Speed constant should go in a new nested class for outtake constants. Motor ID constant should be more clearly labeled and included in the nested class as well.

File: src/main/java/frc/robot/RobotContainer.java [hunk 2]
```diff
@@ -21,10 +24,14 @@
 public class RobotContainer {
   // The robot's subsystems and commands are defined here...
   private final DriveBase driveBase = new DriveBase(0, 1, 2, 3); // TODO: remove dummy port values
+  private final Outtake outtake = new Outtake(Constants.OUTTAKE_ID);
   private final Joystick primaryDriverJoystick =
       new Joystick(IOConstants.PRIMARY_DRIVER_JOYSTICK_PORT);
   private final Joystick secondaryDriverJoystick =
       new Joystick(IOConstants.SECONDARY_DRIVER_JOYSTICK_PORT);
+  private final JoystickButton secondaryDriverButton1 =
```
Comment (aidan-mundy) [inline src/main/java/frc/robot/RobotContainer.java]: Placeholder value should be labeled as a button constant in IOConstants

Rename "secondaryDriverButton1" to "outtakeButton"

**Review by aidan-mundy — CHANGES_REQUESTED** [review]

File: src/main/java/frc/robot/Constants.java [hunk 3]
```diff
@@ -16,15 +16,22 @@
...
     public static final int PRIMARY_DRIVER_JOYSTICK_PORT = 0;
     public static final int SECONDARY_DRIVER_JOYSTICK_PORT = 1;
+    public static final int OUTTAKE_MOTOR_BUTTON_NUMBER = 1; // PLACEHOLDER
+  }
+
+  /**
+   * Stores constants related to the Outtake.
+   */
+  public final class OuttakeConstants {
+    public static final double OUTTAKE_SPEED = 5.0; // PLACEHOLDER
```
Comment (aidan-mundy) [inline src/main/java/frc/robot/Constants.java]: 5.0 is not a valid speed. Speeds for motors go from -1.0 to 1.0

File: src/main/java/frc/robot/Constants.java [hunk 4]
```diff
@@ -16,15 +16,22 @@
...
-  public static final double OUTTAKE_SPEED = 5.0; // PLACEHOLDER
-  public static final int OUTTAKE_ID = 1; // PLACEHOLDER
 
   /**
    * Stores constants related to driver controls, SmartDashboard and other IO (Input/Output).
    */
   public final class IOConstants {
     public static final int PRIMARY_DRIVER_JOYSTICK_PORT = 0;
     public static final int SECONDARY_DRIVER_JOYSTICK_PORT = 1;
+    public static final int OUTTAKE_MOTOR_BUTTON_NUMBER = 1; // PLACEHOLDER
```
Comment (aidan-mundy) [inline src/main/java/frc/robot/Constants.java]:
```suggestion
    public static final int OUTTAKE_BUTTON_NUMBER = 1; // PLACEHOLDER
```

**Review by aidan-mundy — CHANGES_REQUESTED** [review]

**Review by aidan-mundy — APPROVED** [review]

**Review by kchenturtles — APPROVED** [review]
Read through additions. Looks good

**Pull request merged**